source/manual/access-to-licensing-for-third-parties.html.md

```diff
@@ -21,74 +21,131 @@ The source code is hosted on GitHub at [alphagov/licensify](https://github.com/a
-   components.
-   1. [licensify (Staging)](https://deploy.blue.staging.govuk.digital/job/Deploy_App/parambuild?TARGET_APPLICATION=licensify)
-   1. [licensify-admin (Staging)](https://deploy.blue.staging.govuk.digital/job/Deploy_App/parambuild?TARGET_APPLICATION=licensify-admin)
-   1. [licensify-feed (Staging)](https://deploy.blue.staging.govuk.digital/job/Deploy_App/parambuild?TARGET_APPLICATION=licensify-feed)
-1. Manually confirm that the frontend and backend of Licensing are working on Staging before deploying to Production
-1. Follow the same procedure as for staging to deploy to production using the production Jenkins:
-   1. [licensify (Production)](https://deploy.blue.production.govuk.digital/job/Deploy_App/parambuild?TARGET_APPLICATION=licensify)
-   1. [licensify-admin (Production)](https://deploy.blue.production.govuk.digital/job/Deploy_App/parambuild?TARGET_APPLICATION=licensify-admin)
-   1. [licensify-feed (Production)](https://deploy.blue.production.govuk.digital/job/Deploy_App/parambuild?TARGET_APPLICATION=licensify-feed)
+## Using Kubernetes
 
-## Accessing machines using SSH
+The licensing application stack is now orchestrated on Kubernetes clusters (running on AWS EKS). 
 
-Accessing machines using SSH in production and staging requires Production Admin Access. Third parties are usually only given Production Deploy Access, which only allows SSH in the integration environment.
+### Getting Accesss
+To interact with the Kubernetes cluster, you will need to authenticate via the AWS CLI. For more help on how to set up the necessary CLI tools, read [Set up tools to use the GOV.UK Kubernetes platform
+](https://docs.publishing.service.gov.uk/kubernetes/get-started/set-up-tools/).
 
-The machine classes you will need are `licensing_frontend` and `licensing_backend`. You will need to be on the VPN.
+Access to GDS AWS accounts is managed via GDS Users. You can request a user via the [self-service tool](https://gds-request-an-aws-account.cloudapps.digital/) if you have a @digital-cabinet-office.gov.uk email address. Once you have a user, ask a GOV.UK Tech Lead to be given the necessary roles in [govuk-user-reviewer](https://github.com/alphagov/govuk-user-reviewer).
 
-Connect to the instance by running, for example, `gds govuk connect -e production ssh licensing_backend`.
+Once you have the necessary permissions and access, you can continue.
+
+The easiest way to do this is with the GDS CLI. You can either chain your commands onto the GDS CLI:
+```
+gds aws govuk-integration-admin -- aws sts get-caller-identity
+```
+
+...or you can use the `-e` flag with the GDS CLI to export an AWS session into your terminal:
+```
+gds aws govuk-integration-admin -e
+```
+
+If you can't use the GDS CLI, you can use the `aws-vault exec` command with your manually-created AWS tokens instead:
+```
+aws-vault exec govuk-integration -- aws sts get-caller-identity
+```
```

Comment:
It'd be easier to suggest setting up config/credential files using long term creds than using aws-vault?

Reply:
The Re-Engineering stance on this is that if you're authenticating to GDS-owned AWS accounts, you'll have access to the GDS CLI and so probably should be using it - this makes any account or authentication issues easier to troubleshoot.

But we also suggest `aws-vault` as a backup method in case they have any issues using the GDS CLI with their tooling.